file_formats/ie_formats/are_v1.htm

```diff
@@ -1924,8 +2031,13 @@
           </tr>
           <tr>
             <td>0x0048</td>
-            <td>4 (dword)</td>
-            <td>Unused</td>
+            <td>2 (word)</td>
+            <td>Animation width (BGEE)</td>
+          </tr>
+          <tr>
+            <td>0x004a</td>
+            <td>2 (word)</td>
+            <td>Animation height (BGEE)</td>

```

Comment:
Can you note that these two are unused in the other engines?

Reply:
Looks like these values are only needed when WBM or raw PVRZ resources are used for background animations. For regular BAM animations these values are ignored. I'll update the format description accordingly. (Looks like I forgot to add EE-specific flags as well.)